src/components/Button.js

```diff
@@ -58,32 +58,35 @@ class Button extends React.Component {
   };
 
   render () {
-    const theme = StyleUtils.mergeTheme(this.props.theme, this.props.primaryColor);
-    const styles = this.styles(theme);
+    // Manually consume everything that isn't going to be passed down to the button so we don't have to keep adding props one at a time.
+    // Keep elementProps for backwards compatibility.
+    const { actionText, buttonRef, children, elementProps, icon, isActive, primaryColor, style, theme, ...rest } = this.props
+    const mergedTheme = StyleUtils.mergeTheme(theme, primaryColor);
+    const styles = this.styles(mergedTheme);
 
     return (
       <button
-        aria-label={this.props['aria-label']}
-        onClick={this.props.type === 'disabled' ? null : this.props.onClick}
```

Comment:
Does removing this mean that setting the `disabled` attribute to `true` will prevent the `onClick` handler from firing?

Reply:
I answered this in a slack message but the answer is yes.